docs/source/user_guide/installation/GNU-linux.rst

```diff
@@ -121,10 +119,10 @@ order to use the wrapped SUNDIALS ODE and DAE
 
    .. code:: bash
 
-	   apt install libopenblas-dev
-	   pybamm_install_odes
+      apt install libopenblas-dev
+      pybamm_install_odes
 
-   The ``pybamm_install_odes`` command is installed with PyBaMM. It automatically downloads and installs the SUNDIALS library on your
+   The ``pybamm_install_odes`` command, installed with PyBaMM, automatically downloads and installs the SUNDIALS library on your
```

Comment:
```suggestion
```

This line is present below as well, we might as well keep it in one place

Reply:
Ah yes, i'll also fix it's indentation to appear on both tabs.